en/tutorial/nixos-install-with-disko.md

```diff
@@ -0,0 +1,176 @@
+# Install NixOS and partition the disk with `disko`
+
+This tutorial adds another layer of reproducibility to the [[nixos-install]] by writing your partitioning scheme for #[[nixos]] in #[[nix]].
+
+>[!note] Flake configuration
+> We will use #[[flakes]] by default.
+
+{#iso}
+## Install NixOS
+
+>[!note] Minimal ISO image
+> This tutorial doesn't use a graphical installer. Instead, it uses the minimal ISO image. This is primarily because we don't want the installer to partion the disk for us. We will use [disko](https://github.com/nix-community/disko) to do that.
+
+- Download the latest NixOS ISO from [here](https://nixos.org/download#download-nixos). Choose the "Minimal ISO image" for your architecture.
+- Create a bootable USB flash drive ([instructions here](https://nixos.org/manual/nixos/stable/index.html#sec-booting-from-usb)) and boot the computer from it.
+
+NixOS will boot into the USB in CLI mode. 
+
+:::{.center}
+![[nixos-installer-cli.jpeg]]
+:::
+
+{#partition}
+## Partition the disk
+
+The [disko quickstart guide](https://github.com/nix-community/disko/blob/master/docs/quickstart.md) does an excellent job of explaining it. We will follow the same steps and include screenshots wherever necessary. Additionally, in the last step we will use flakes to manage the configuration.
+
+### Choosing the disk configuration
+
+Disko provides a few examples to choose [from](https://github.com/nix-community/disko/tree/master/example). We will use the [hybrid](https://github.com/nix-community/disko/blob/master/example/hybrid.nix) example as it will work for both BIOS and UEFI systems.
+
+Copy the disk configuration on to the USB flash drive.
```

Comment:
Are we really copying to this the USB flash drive?

Reply:
Yes, the `/tmp/disko-config.nix` exists on USB flash drive, if it didn't, the config wouldn't persist on partitioning.

Reply:
Referring to as `/tmp` seems to be more appropriate